Gameplay/entities/enemies/BossController.cpp

```diff
@@ -315,22 +323,34 @@ float Hachiko::Scripting::BossController::GetPlayerDistance()
 
 void Hachiko::Scripting::BossController::WaitingController()
 {
+    // Can be used to start boss by proximity, it is currently started via a hit
```

Comment:
We can delete this now that we have the new design. If necessary we can always check the commit history.

Reply:
Yep ur right we shouldnt leave it here

Reply:
Let's remove it and then merge 👍